src/Foundations/GridSystem.mdx

```diff
@@ -129,3 +179,64 @@ Use Clearfix and set to visible for the viewports with columning issues, so that
 }}
 
 </Playground>
+
+## Props
```

Comment:
`docz` has a built-in component that will auto-generate props documentation from the components, though it may not print it as nicely as you have here. For more, see: https://www.docz.site/docs/built-in-components#component-props

Reply:
Oh HAHA you tell me this after I did it all manually! I'll take a loot.

Reply:
Removing this Props section from this PR since it wasn't in scope to begin with. Getting the Props component working in docz isn't straight forward and might require changes to core components.